clusterDeployer.py

```diff
@@ -50,8 +50,9 @@ def __init__(self, cc: ClustersConfig, ai: AssistedClientAutomation, steps: List
 
         lh = host.LocalHost()
         lh_config = list(filter(lambda hc: hc.name == lh.hostname(), self._cc.hosts))[0]
-        self._local_host = ClusterHost(lh, lh_config, cc, cc.local_bridge_config)
-        self._remote_hosts = {bm.name: ClusterHost(host.RemoteHost(bm.name), bm, cc, cc.remote_bridge_config) for bm in self._cc.hosts if bm.name != lh.hostname()}
+        serves_dhcp = cc.kind != "microshift" and len(cc.local_vms()) != len(cc.all_nodes())
```

Comment:
Small nit, might be better to check `kind == "openshift"` we were discussing adding additional cluster "kinds" aside from openshift / microshift.

Reply:
Ack.